code_schemes/rqas/aik/aik_political_participation.json

```diff
@@ -13,6 +13,38 @@
             "VisibleInCoda": true,
             "Shortcut": "e"
         },
+        {
+            "CodeID": "code-4f76dcad",
+            "CodeType": "Normal",
+            "DisplayText": "yes, no reason",
+            "NumericValue": 1,
+            "StringValue": "yes_no_reason",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-9b74bd5a",
+            "CodeType": "Normal",
+            "DisplayText": "no, no reason",
+            "NumericValue": 2,
+            "StringValue": "no_no_reason",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-1427d73a",
+            "CodeType": "Normal",
+            "DisplayText": "maybe",
```

Comment:
should this be "maybe" or "i do not know" for consistency?
A question for RDA, as the meaning is not exactly the same

Reply:
I've been advised to leave it as it is.